### PR TITLE
Residential and commercial floor space.

### PR DESCRIPTION
**Pull request by robinhasse**
Instead of filtering the total floor space, we now write residential, commercial and total floor space into `p36_floorspace.cs4r`. This allows to report subsector floor space in remind2. See [#379](https://github.com/pik-piam/remind2/pull/379) for those changes.

### CODE REVIEW

File: R/calcFloorspace.R [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+#' Floor space in buildings
+#'
+#' Residential, commercial and total floor space fro EDGE-B
+#' @importFrom madrat readSource
+#' @author Antoine Levesque, Robin Hasse
+#' @return MAgPIE object with buildings floor space
 calcFloorspace <- function() {
-  
-  data <- readSource("EDGE", subtype = "Floorspace") 
-  data <- collapseNames(data[,,"buildings"])
```
[the remark this answers is not in the document]
Comment (robinhasse) [reply R/calcFloorspace.R:5]: I se the point and adding this is no effort. On the other hand, using newer input data will be impossible in many occasions. We just broke this option with the introduction of a new scenario that is now part of the input data and will give you a Domain violation Error with any REMIND version that is more than one week old.

[the remark this answers is not in the document]
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply R/calcFloorspace.R:5]: Ahh, wasn't aware of that.

**Review by LaviniaBaumstark — APPROVED** [review]
merge needs to be coordinated with changes in REMIND

File: R/calcFloorspace.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (robinhasse) [reply R/calcFloorspace.R:5]: So after some discussion some time ago also with @LaviniaBaumstark, I decided to create a new file `f36_floorspace.cs4r` with the sub-sectoral floor space and keep the old `p36_floorspace.cs4r` for a while. Once input data eith the new file is available, I will adapt REMIND and remind 2 to make use of it. This PR can be merged with no side-effects and compatibility issues expected.

**Pull request merged**